documentation/content/en/articles/committers-guide/_index.adoc

```diff
@@ -3075,6 +3075,23 @@ Relnotes:	yes
 
 ====
 
+==== Group and team names
+
+[cols="1,1"]
+|===
+|Group name in Phabricator |Team name for commit
+
+|Ports Committers
```

Comment:
`git -C /usr/ports log --no-expand-tabs --extended-regexp --grep='Approved by:[[:cntrl:] ]{1,}port'`

* finds approver IDs that begin with the string `port`
* is not enough for me to guess the team name that corresponds to `Ports Committers`.

Reply:
Obscurely, I discovered the likely answer through guesswork plus misuse of a field in Bugzilla: 

![image](https://user-images.githubusercontent.com/192271/208510422-f74a4061-3aeb-4c5e-8c1c-c98a23a69fd9.png)